spec/lib/parser_spec.rb

```diff
@@ -19,4 +19,30 @@
       end
     end
   end
+
+  describe "#count_lines" do
+    subject { described_class.new("./spec/fixtures/games_test.log") }
+
+    it "returns the number of lines in the file" do
+      expect(subject.count_lines).to eq(158)
+    end
+
+    #verificar com os avaliadores/reviwers qual modo usar para testar o count_lines (de cima ou de baixo)
```

Comment:
Se for seguir nessa abordagem eu faria o de cima, pra manter a descrição e o seu `subject` ser sempre a instancia e nao a execução do metodo. Mas da pra fazer da segunda forma tbm, eu faria assim:

```rb
describe "#count_lines" do
  subject { described_class.new(file_path).count_lines }

  let(:file_path) { "./spec/fixtures/games_test.log" }

  it { is_expected.to eq(158) }
end
```

https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax